ui/branding/__init__.py

```diff
@@ -22,7 +22,12 @@ def init_menu():
             "documents.create",
             _("New upload"),
             order=1,
+            # TODO: create function that checks if user has any permissions and as optimalization cache it in session
+            # visible_when=check_permissions,
         )
+        # hide the /admi (maximum recursion depth exceeded menu)
+        admin_menu = current_menu.submenu("settings.admin")
```

Comment:
Could we move this to oarepo-ui please? As this is broken on every repository app